jobs/postfix/spec

```diff
@@ -43,3 +51,14 @@ properties:
     tls_key:
       description: "TLS Key"
       default: ""
+    sasl_users:
+      description: "list of users and passwords used for SASL auth"
+      default:
```

Comment:
Shouldn't have `default` users. Use an `example` instead.

Reply:
Ah.  Did not know that was a possibility.  Will fix that.